packages/bundler/src/runBundler.ts

```diff
@@ -74,15 +74,16 @@ export async function runBundler (argv: string[], overrideExit = true): Promise<
     .option('--network <string>', 'network name or url')
     .option('--mnemonic <file>', 'mnemonic/private-key file of signer account')
     .option('--entryPoint <string>', 'address of the supported EntryPoint contract')
-    .option('--port <number>', `server listening port (default: ${bundlerConfigDefault.port})`)
+    .option('--publicApiPort <number>', `server listening port for public clients (default: ${bundlerConfigDefault.publicApiPort})`)
+    .option('--privateApiPort <number>', `server listening port for block builders (default: ${bundlerConfigDefault.privateApiPort})`)
     .option('--config <string>', 'path to config file', CONFIG_FILE_NAME)
     .option('--auto', 'automatic bundling (bypass config.autoBundleMempoolSize)', false)
     .option('--unsafe', 'UNSAFE mode: no storage or opcode checks (safe mode requires geth)')
     .option('--debugRpc', 'enable debug rpc methods (auto-enabled for test node')
     .option('--conditionalRpc', 'Use eth_sendRawTransactionConditional RPC)')
     .option('--show-stack-traces', 'Show stack traces.')
     .option('--createMnemonic <file>', 'create the mnemonic file')
-    .option('--useRip7560Mode', 'Use this bundler for RIP-7560 node instead of ERC-4337 (experimental).')
+    .option('--useRip7560Mode <string>', 'Use this bundler for RIP-7560 node instead of ERC-4337 (experimental).')
```

Comment:
our bundler already have "--dev" mode. 
the userRip7560Mode should stay as boolean. push mode should be  triggered by "--dev"

Reply:
Why?